### ISSUE TITLE
System info exception prevents game from launching

### ISSUE THREAD

**Issue by Shished** [Heroic-Games-Launcher/HeroicGamesLauncher#3051]
### Describe the bug

After pressing the play button in the UI the game's status changes to "Launching" and stays this way until the HGL is restarted, the game does not starts. All installed games fails to launch.
Used Flatpak build of HGL.

### Add logs

```text
(17:01:15) INFO:    [ProtocolHandler]:  received 'heroic://launch/c38f51843fdf4db0b24fc907b8d78221'
(17:01:15) INFO:    [Backend]:          Launching DEATH STRANDING DIRECTOR'S CUT (c38f51843fdf4db0b24fc907b8d78221)
(17:01:15) INFO:    [Backend]:          Downloading saves for DEATH STRANDING DIRECTOR'S CUT
(17:01:15) INFO:    [Legendary]:        Syncing saves for DEATH STRANDING DIRECTOR'S CUT: XDG_CONFIG_HOME=/home/redacted/.var/app/com.heroicgameslauncher.hgl/config/heroic/legendaryConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary sync-saves c38f51843fdf4db0b24fc907b8d78221 --skip-upload --save-path "/disk/Heroic/Prefixes/DEATH STRANDING DIRECTORS CUT/drive_c/users/redacted/AppData/LocalLow/KojimaProductions/DeathStrandingDC/0964ef10fe2d4062ac25d5645fd358df" -y
(17:01:15) INFO:    [Backend]:          Preventing machine to sleep
(17:01:20) INFO:    [Legendary]:        installed.json updated, refreshing library
(17:01:20) INFO:    [Legendary]:        installed.json updated, refreshing library
(17:01:20) INFO:    [Backend]:          Saves for DEATH STRANDING DIRECTOR'S CUT downloaded
(17:01:20) INFO:    [Backend]:          Preventing display from sleep
(17:01:20) INFO:    [Legendary]:        Running command: XDG_CONFIG_HOME=/home/redacted/.var/app/com.heroicgameslauncher.hgl/config/heroic/legendaryConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(17:01:20) INFO:    [Gog]:              Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/redacted/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json --version
(17:01:20) INFO:    [Nile]:             Running command: XDG_CONFIG_HOME=/home/redacted/.var/app/com.heroicgameslauncher.hgl/config/heroic/nile_config /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/nile --version
Error occurred in handler for 'launch': TypeError: Invalid number
    at filesize (/app/bin/heroic/resources/app.asar/build/electron/main.eeeeb87e.js:108:1282)
    at getSystemInfo (/app/bin/heroic/resources/app.asar/build/electron/main.eeeeb87e.js:125:3270)
    at async /app/bin/heroic/resources/app.asar/build/electron/main.eeeeb87e.js:191:10841
    at async node:electron/js2c/browser_init:2:98394
```

### Steps to reproduce

1. Open HGL
2. Press play button on any installed game in the library

### Expected behavior

The game will start.

### Screenshots

![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/3143668/f5d013f7-d5b5-42b5-9e3c-ca1de66e3c05)

### Heroic Version

Latest Stable (Flatpak)

### System Information

os_release: 
    name:               "Arch Linux"
lspci: 
    03:00.0 VGA compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] Navi 22 [Radeon RX 6700/6700 XT/6750 XT / 6800M/6850M XT] (rev c5)
glxinfo: 
    gpu:                AMD Radeon RX 6700 XT (navi22, LLVM 16.0.6, DRM 3.54, 6.5.3-zen1-1-zen) (0x73df)
    gpu_version:        23.1.7
lscpu: 
    model_name:         Intel(R) Core(TM) i5-12600
xdg_current_desktop:    GNOME
desktop_session:        gnome

### Additional information

HGL log does not captures the error on play press, the log output was printed into the terminal.

**Comment by imLinguin** [comment]
Hey, I suspect the new system info utility is unable to read memory usage property. 
Could you try launching heroic with `LANG=C` env var to confirm this theory?

Full command would be `LANG=C flatpak run com.heroicgameslauncher.hgl`

**Comment by Shished** [comment]
Yes, this fixes the problem.
I ran this command: `flatpak run --env=LANG=C com.heroicgameslauncher.hgl`.

**Comment by lgdsgd2000** [comment]
I also have this problem

**Comment by lgdsgd2000** [comment]
But LANG=C is useless to me, the game still won't start

**Comment by imLinguin** [comment]
@lgdsgd2000 are you sure you have the same issue? Could you provide any logs to confirm?

**Comment by lgdsgd2000** [comment]
```
(22:36:06) INFO:    [Gog]:              Getting game metadata: /opt/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/djw/.config/heroic/gog_store/auth.json info 1495134320 --token <redacted> --lang=en-US --os windows
(22:36:06) INFO:    [Gog]:              Getting game metadata: /opt/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/djw/.config/heroic/gog_store/auth.json info 1495134320 --token <redacted> --lang=en-US --os windows
Error occurred in handler for 'launch': TypeError: Invalid number
    at filesize (/opt/heroic/resources/app.asar/build/electron/main.c2fc9376.js:108:1282)
    at getSystemInfo (/opt/heroic/resources/app.asar/build/electron/main.c2fc9376.js:125:3270)
    at async /opt/heroic/resources/app.asar/build/electron/main.c2fc9376.js:191:10841
    at async node:electron/js2c/browser_init:2:98394
```

**Comment by lgdsgd2000** [comment]
The game's status also changes to "Launching" but won't start. When I click the System Information, this log was printed in terminal
```
Error occurred in handler for 'getSystemInfo': TypeError: Invalid number
    at filesize (/opt/heroic/resources/app.asar/build/electron/main.c2fc9376.js:108:1282)
    at getSystemInfo (/opt/heroic/resources/app.asar/build/electron/main.c2fc9376.js:125:3270)
    at async node:electron/js2c/browser_init:2:98394

```
Same problem even if I start HGL with LANG=C.

**Comment by CommandMC** [comment]
@lgdsgd2000 Could you share the output of `LANG=C free -b`? Heroic runs this command to get the used & free amount of RAM on your system, seems like that's going wrong somewhere.
#3070 should "fix" the error (by simply reporting 0/0MB RAM used if something *does* go wrong), but we'd of course like to actually report correct data if possible

**Comment by jexjws** [comment]
My system is manjaro. I ran the `free -b` command on konsole and the response was the error `free: Multiple unit options don't make sense.`. I googled the response and found this :

https://forum.manjaro.org/t/free-multiple-unit-options-doesnt-make-sense/98741

Then I opened .zshrc and found /usr/share/zsh/manjaro-zsh-config, and then I commented out the line alias free='free -m'. Now, the `free -b` command works fine.

**Comment by jexjws** [comment]
@lgdsgd2000 

Your output command is in Chinese, so you probably didn’t add `LANG=C` , please try `LANG=C free -b`，after confirming that the output of `LANG=C free -b` does not contain Chinese, try `flatpak run --env=LANG=C com.heroicgameslauncher.hgl`

###### 试试看 `LANG=C free -b` 确认下（你的输出指令带中文，或许没有运行前设定环境变量LANG=C），输出应该不带中文的。确认了`LANG=C free -b`输出不带中文后，再尝试下`flatpak run --env=LANG=C com.heroicgameslauncher.hgl`

**Comment by lgdsgd2000** [comment]
@jexjws 都会带中文，应该是konsole的问题，换git包了，楼上说了有个pull处理了一下这个问题，也就懒得管了，能用就行

**Comment by Alllgator343** [comment]
Hello. I have the same problem on Linux Mint xfce 21.2.

HGL 2.9.2 (Deb)

`Error occurred in handler for 'launch': TypeError: Invalid number
    at filesize (/opt/Heroic/resources/app.asar/build/electron/main.c2fc9376.js:108:1282)
    at getSystemInfo (/opt/Heroic/resources/app.asar/build/electron/main.c2fc9376.js:125:3270)
    at async /opt/Heroic/resources/app.asar/build/electron/main.c2fc9376.js:191:10841
    at async node:electron/js2c/browser_init:2:98394
`

**Comment by DmytroKmetsynskyi** [comment]
same issue

**Comment by dankxylese** [comment]
Bumping as I have the same issue. I have a non-flatpack install, and running the following fixed my issues:
LANG=C /opt/heroic/heroic

OS: ArchLinux 
Kernel: 6.5.9-arch2-1 
Packages: 1640 (pacman), 18 (flatpak) 
Shell: bash 5.2.15 
DE: Plasma 5.27.9 
WM: KWin 
CPU: AMD Ryzen 5 3600 (12) @ 3.600GHz 
GPU: NVIDIA GeForce RTX 2070 SUPER 
Memory: 11.34GiB / 15.56GiB (72%) 
GPU Driver: NVIDIA 535.113.01 
CPU Usage: 17% 
Disk (/): 741G / 923G (81%)

**Comment by CommandMC** [comment]
This issue is resolved on main, the fix will thus be included in the next Heroic release

**Comment by arielj** [comment]
this should be fixed now in 2.10.0
